sentinel-demo/sentinel-demo-command-handler/pom.xml

```diff
@@ -16,5 +16,9 @@
             <groupId>com.alibaba.csp</groupId>
             <artifactId>sentinel-transport-simple-http</artifactId>
         </dependency>
+        <dependency>
```

Comment:
the `sentinel-transport-simple-http` contains the  `sentinel-transport-common`. I think this import is unnecessary. do you think so?

Reply:
> the `sentinel-transport-simple-http` contains the `sentinel-transport-common`. I think this import is unnecessary. do you think so?

I think implementation layer dependencies may use "provide" scope. Of cause, it can also be removed here

Reply:
I have removed, please review